strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/RetryDownloadArtifactWithPermanentFailureStartingAtSomePointTest.java

```diff
@@ -34,7 +34,7 @@
     private PermanentBrokenArtifactInputStream brokenArtifactInputStream;
 
     @Before
-    public void timeoutRetryFeatureRatherQuicklyForTestPurposes()
+    public void setup()
```

Comment:
BTW, this tests fails for me in Eclipse with following strange error:
```
java.lang.IllegalAccessException: Class org.junit.runners.model.FrameworkMethod$1 can not access a member of class org.carlspring.strongbox.providers.repository.RetryDownloadArtifactTestBase with modifiers "public"
```
do you have it in Idea? 
If not maybe you have any thoughts why it's not working in Eclipse?

Reply:
> do you have it in Idea?

No I don't , sorry :(

> If not maybe you have any thoughts why it's not working in Eclipse?

No idea :(